### DISCUSSION TITLE
Usage doubt

### DISCUSSION THREAD

**Discussion by forloopcodes** [aayush-tripathi/JEEPYQREPLYBOT#27] [Q&A] [answered]
Bro how can i use it to get particular chapter question, i need a whole database for my app

**Comment by sujaldev** [comment, marked as answer]
Check the subjects directory, there isn't a lot there because the plan was to have users generate a database.

**Reply by forloopcodes** [reply]
OMG BROOO I WAS WORRIED ABOUT PYQ AND ALL IN MY APP, AND YOU GAVE BEST IDEA TYSM, i can just make users add more questions and they get reward points per question awesome
